joplin/api/schema.py

```diff
@@ -173,6 +174,9 @@ class Meta:
     def resolve_janis_urls(self, info):
         return self.specific.janis_urls()
 
+    def resolve_page_type(self, info):
+        return self.content_type
+
```

Comment:
@briaguya I took a look at what you were referencing yesterday. 
- With added this here, I was able to get a reference of the "content_type"/"page_type" for each page within the allPages query (link in pr description). 
-  Also, I was able to remove the page_type from the Location and Event pages.


NOTE: But, why call it `page_type` now...? I guess it should just be called `content_type` if we go this way???